Signal/src/call/CallService.swift

```diff
@@ -1114,6 +1156,10 @@ protocol CallServiceObserver: class {
                 }
             }
         }
+
+        // Prevent screen from dimming during video call.
+        let hasLocalOrRemoteVideo = localVideoTrack != nil || remoteVideoTrack != nil
+        UIApplication.shared.isIdleTimerDisabled = hasLocalOrRemoteVideo
```

Comment:
I wondered about that.  The documentation doesn't indicate that to that affect, but it _is_ part of UIKit and UIKit main-thread only.  I'm switching to be safe. Thanks!